### DISCUSSION TITLE
Property getStyleSet not exist when trying to add static geojson to map.

### DISCUSSION THREAD

**Discussion by sunyonglincanada** [heremaps/harp.gl#2098] [Q&A] [answered]
Hi there,

I was trying to follow the tutorial to add static geojson to the map - https://imgur.com/a/Acn54tA but got an error that Property getStyleSet not exists. Here is the screenshot of the error - https://imgur.com/a/MDvXk5O

I've updated my code in the repository for the reference - https://github.com/sunyonglincanada/harp.gl-GLTF

Can anyone take a look to see if there is any solution on it? Thanks in advance!!

**Comment by nzjony** [comment, marked as answer]
I made a PR: https://github.com/sunyonglincanada/harp.gl-GLTF/pull/4

I fixed the documentation, see: https://developer.here.com/tutorials/harpgl/, thanks a lot @sunyonglincanada !!

**Reply by sunyonglincanada** [reply]
Thanks @nzjony for the updates.  I will merge your PR and try it on my end.
